src/components/apiErrorAlert/apiErrorCode.ts

```diff
@@ -135,6 +139,7 @@ export enum ApiErrorCode {
     UGYLDIG_FØDSELSNUMMER = 'ugyldig_fødselsnummer',
     UGYLDIG_GRUNN_FOR_UNDERKJENNING = 'ugyldig_grunn_for_underkjenning',
     UGYLDIG_INPUT = 'ugyldig_input',
+    UGYLDIG_JOURNALPOSTID = 'ugyldig_journalpostId',
```

Comment:
```suggestion
    UGYLDIG_JOURNALPOSTID = 'ugyldig_journalpost_id',
```
Vi bruker vel ikke camel-case for errorCodes? Må evt. endres bakover og